legacy_tester/parsed_json_test_definitions.py

```diff
@@ -3,5 +3,26 @@ def phase_one_regions_parse_not_included_countries(published_json, test_payload)
     for to_be_excluded_countries in test_payload:
         if to_be_excluded_countries in parsed_countries:
             print('Error: {} found in list of phase 1 parsed countries'.format(to_be_excluded_countries))
-            return
-    print("PASS")
\ No newline at end of file
+            return False
+    return True
+
+
+def unit_of_subtype_of_country_should_contain_exactly_x_tonals_remarks_ending_with_y(published_json, test_payload):
+    for unit_payload in test_payload:
+        country_id_to_check = next(filter(lambda a: a.country == unit_payload['country'],
```

Comment:
I _think_ we can just check the contents of the tonals table, without worrying about country, platform_sub_type, unit_title.   Then the values for these entities in the mock data can change, without breaking our test.

Reply:
We can change that. But I believe this will be a more specific and stringent test.

I can add a new test just to check just the tonals with these remarks. Shall I?

Reply:
The reason for relaxing the text, is that I want to minimise the effort we spend on maintenance of tests as the mock data matures.  `Narnia` may get renamed to `Wales` - but if there are 3 matching tonals in the whole import, things are still ok.  If `Narnia` get duplicated, then the `3` will go to `6`. The reason should be clear, and we just update the test to `6`.

Reply:
Okay, let me relax it.

Reply:
Please check now.